pkg/model/config/config.go

```diff
@@ -45,7 +45,7 @@ type Config struct {
 
 	// Resources tracks scaffolded resources in the project
 	// This info is tracked only in project with version 2
-	Resources []GVK `json:"resources,omitempty"`
+	Resources []Data `json:"resources,omitempty"`
```

Comment:
Can we call it `ResourceData`? `Resource` collided with the other type but `Data` seems pretty generic.
```suggestion
	Resources []ResourceData `json:"resources,omitempty"`
```

Reply:
It was resource data before. However, you suggested it be changed for Data. But it is fine, done 👍

Reply:
Before, in https://github.com/kubernetes-sigs/kubebuilder/pull/1696#discussion_r528867742, I just meant to change to `Data` the method mentioned in https://github.com/kubernetes-sigs/kubebuilder/pull/1696#discussion_r530974252, but I guess I wasn't clear enough. I think that it is clear now. `ResourceData` for the type, `Data` for the method as the `Resource` is implied.